extras/jenkins/Minikube/Jenkinsfile

```diff
@@ -105,7 +105,7 @@ pipeline {
             curl -Lo minikube https://storage.googleapis.com/minikube/releases/latest/minikube-linux-amd64  && chmod +x minikube
             mkdir -p /usr/local/bin/
             install minikube /usr/local/bin/
-            minikube start --vm-driver=docker --force --cpus 4 --memory 30000
+            minikube start --vm-driver=docker --force --cpus 4 --memory 30000 --kubernetes-version=v1.23.9
```

Comment:
Went back and forth on this; I think this needs to be a variable. However, I think that the right thing to do is to make it a variable when it's kicked off by runner - otherwise we introduce two new Jenkins variables that we need to keep track of. This is a known working config for now. Raised #190 to handle this.